Gemfile

```diff
@@ -1,8 +1,14 @@
 source 'https://rubygems.org'
 
-group :test do
-  gem 'rspec'
-  gem 'rubocop', '0.79.0'
-  gem 'simplecov', require: false
+group :test, :development do
+  gem "sinatra"
+  gem "capybara", require: false, group: :test
+  gem "selenium-webdriver"
+  gem "rspec", require: false, group: :test
```

Comment:
nice, is this a way of specifying that capybara and rspec should be only for the test group and not the development group?